### DISCUSSION TITLE
How to find the right reader for my local Data

### DISCUSSION THREAD

**Discussion by Nebelgeist** [OpenDrift/opendrift#1051] [Q&A]
Hi there,

I am a student and inexperienced in programming, so I may have simply overlooked how this works. Can you tell me how I can use a reader to read in a local data set (netCDF format) with various data on currents and ice?
more precisely:  
 - sea_ice_area_fraction
 - northward_sea_water_velocity
 - eastward_sea_water_velocity
 - eastward_sea_ice_velocity
 - sea_ice_thickness
 - northward_sea_ice_velocity
 - sea_surface_height_above_geoid
 - y_sea_water_velocity
 - x_sea_water_velocity

I tried reader_netCDF_CF_generic and got the following messages:

> INFO    opendrift.readers.reader_netCDF_CF_generic:129: Opening dataset: CMEMS-GLOBAL_001_030-several_vars-1994.nc
> DEBUG   opendrift.readers.reader_netCDF_CF_generic:146: Finding coordinate variables.
> INFO    opendrift.readers.reader_netCDF_CF_generic:285: Grid coordinates are detected, but proj4 string not given: assuming latlong
> DEBUG   opendrift.readers.basereader.variables:546: Setting buffer size 1380 for reader CMEMS-GLOBAL_001_030-several_vars-1994.nc, assuming a maximum average speed of 5 m/s and time span of 29 days, 12:00:00
> INFO    opendrift.readers.basereader:158: Variable y_sea_water_velocity will be rotated from northward_sea_water_velocity
> INFO    opendrift.readers.basereader:158: Variable x_sea_water_velocity will be rotated from eastward_sea_water_velocity
> DEBUG   opendrift.readers.basereader:168: Variable mapping: ['sea_floor_depth_below_sea_level'] -> ['land_binary_mask'] is not activated.
> 

Does someone have an idea about this?

**Comment by knutfrode** [comment]
Seems you never got any answer to this?

Anyway, these log messages are normal, and does not indicate any errors. So this reader can be added to a simulation object, like in all examples in the gallery.

If you want to access the data without running an OpenDrift simulation, you might be better off by simply using Xarray.

**Comment by Nebelgeist** [comment]
Thank you very much for the answer.

I think the problem was in my data set. Now the simulation is running.

**Comment by knutfrode** [comment]
Very good, then closing this issue.
